dbms/src/Storages/DeltaMerge/File/DMFilePackFilter.h

```diff
@@ -193,8 +193,10 @@ class DMFilePackFilter
                 tryLoadIndex(attr.col_id);
             }
 
+            Stopwatch watch;
             const auto check_results = filter->roughCheck(0, pack_count, param);
             std::transform(use_packs.begin(), use_packs.end(), check_results.begin(), use_packs.begin(), [](UInt8 a, RSResult b) { return (static_cast<bool>(a)) && (b != None); });
+            scan_context->total_dmfile_rough_set_index_load_time_ns += watch.elapsed();
```

Comment:
rename to `total_dmfile_rough_set_index_check_time_ns` ?

Reply:
yes, we can distinguish rough set index `loading time` and `checking time`

Reply:
Does this mean that we need to use two indicators separately for statistics?

Reply:
I think the sum is enough.